HHS/Head-Start-TTADP
# [TTAHUB-1862] Bug fix with removing recipients

## Description of change

ObjectiveFiles, despite being marked "onDelete: CASCADE" in the model definition, are blocking the removal of recipients, throwing an error when that recipients objectives are removed, causing the entire transaction to be rolled back

## How to test

Fully create an activity report with many recipients, adding one goal and objective with files. Confirm everything is saved properly. To view bug, you can remove a recipient while on ```main``` branch. You should see a 500 error when saving after doing so. On this branch, there is no such error. (You can also comment out lines 1703 - 1709 of /services/goals.js and run the test I added)


## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1862


## Checklists

### Every PR



### Production Deploy


### After merge/deploy
